### ISSUE TITLE
connecting mysql

### ISSUE THREAD

**Issue by heygambo** [l3iggs/docker-LAMP#5]
How can I connect to the mysql db using a client like Sequal Pro?
It always says the host doesn't answer.

**Comment by iamvitali** [comment]
@greyltc Same thing, trying to connect to 127.0.0.1:3306 from my machine running the docker and it just says `MySQL said: Can't connect to MySQL server on '127.0.0.1' (61)`.

**Comment by greyltc** [comment]
Yeah, there is something wrong all of a sudden here with the mysql server. It silently doesn't start with no error output anywhere. I've got a workaround in place for now. Let me know if the latest image fixes the issue.

**Comment by kaloczidev** [comment]
Hey, any update on this?

**Comment by slabruyere** [comment]
I haven't found a way to connect to the DB from the host yet, but the MySQL is definitely running in the container.
I just ran `docker exec -it {container-hash} /bin/bash` and then `mysql -u root` and entered nothing when prompt asked for password and created a DB by simply entering `CREATE DATABASE my_database`.

**Comment by jccpdev** [comment]
I ran into the same problem, What I did to get around it is change the host of the root user... with this `mysql -u root -e "SELECT User,Host from mysql.user"` (Inside the docker box) (I am using this for development... this IS NOT a good command to run if your using in production). But @greyltc maybe the Variables for the mysql user can be set ad ENV variables. I am not familiar enough with docker to do it my self. If not I would have sent you a PR. But it may help.
